modules/mongodb/collect_test.go

```diff
@@ -0,0 +1,116 @@
+package mongo
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/netdata/go.d.plugin/agent/module"
+	v5_0_0 "github.com/netdata/go.d.plugin/modules/mongodb/testdata/v5.0.0"
```

Comment:
The package name is `v5_0_0`, need for an alias here.

Reply:
done